anthropic/actions/code.ts

```diff
@@ -0,0 +1,26 @@
+import { shortcircuit } from "deco/engine/errors.ts";
+import { AppContext } from "../mod.ts";
+import { Anthropic } from "../deps.ts";
+export interface Props {
+  system?: string;
+  messages: Anthropic.MessageParam[];
+}
+
+export default async function chat(
+  { system, messages }: Props,
+  _req: Request,
+  ctx: AppContext,
+) {
+  if (!messages) {
+    return shortcircuit(new Response("No messages provided", { status: 400 }));
+  }
+
+  const msg = await ctx.anthropic.messages.create({
+    system,
+    model: "claude-3-opus-20240229",
```

Comment:
Should it be passed as a prop? And have all the options on the type?

Reply:
Agree!